### DISCUSSION TITLE
Should emulation notebooks be Part 3?

### DISCUSSION THREAD

**Discussion by kmarkovich** [gmdsi/GMDSI_notebooks#147] [Ideas]
I am going to work up a response matrix emulation notebook (based off of Jeremy and Rui's super cool recent work) and it occurred to me that it should probably be part of a subgenre of notebooks covering the various emulation approaches (Rui's DSI notebooks, Jeremy's GPR prototype, etc.). Do y'all think Part 3 could be focused on emulation? Or should emulator notebooks still be in Part 2? @mnfienen  @jtwhite79 @briochh @rhugman

**Comment by jtwhite79** [comment]
Awesome Katie.  I think emulation holds a lot promise and could see it being an advanced part3....maybe?  Others?

**Comment by rhugman** [comment]
Thanks Katie! I am not too fussed either way, but don't think I would go for a Part3. I see emulation as just another tool in the "applied workflows". But happy to go with whatever y'all prefer...
